contracts/LifCrowdsale.sol

```diff
@@ -334,6 +343,8 @@ contract LifCrowdsale is Ownable, Pausable {
 
       foundationWallet.transfer(this.balance);
 
+      mintExtraTokens(uint256(12));
```

Comment:
I think this should be 24 instead of 12. When there is no MVM, the vesting schedule for the 5% is still 2 years starting in 2 years

Reply:
:+1: